### ISSUE TITLE
Autocomplete suggestions are no longer read with NVDA

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#234465]
Type: <b>Bug</b>

With NVDA, type something that triggers autocomplete.

Notice that nothing is announced when the menu appears. Pressing up/down to go through autocomplete suggestions also doesn't read anything.

This worked yesterday, so this is a very recent regression.

VS Code version: Code - Insiders 1.96.0-insider (90868576241dd25c6c5da64adadc0a09de91a9fe, 2024-11-22T09:56:24.579Z)
OS version: Windows_NT x64 10.0.22631
Modes:
Remote OS version: Linux x64 5.15.167.4-microsoft-standard-WSL2

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 PRO 7840U w/ Radeon 780M Graphics   (16 x 3294)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|62.72GB (47.27GB free)|
|Process Argv|--folder-uri=vscode-remote://wsl+ubuntu/home/dickson/source/leetcode-2024 --remote=wsl+ubuntu|
|Screen Reader|yes|
|VM|40%|

|Item|Value|
|---|---|
|Remote|WSL: ubuntu|
|OS|Linux x64 5.15.167.4-microsoft-standard-WSL2|
|CPUs|AMD Ryzen 7 PRO 7840U w/ Radeon 780M Graphics (16 x 0)|
|Memory (System)|30.72GB (29.09GB free)|
|VM|0%|
</details><details><summary>Extensions (17)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-containers|ms-|0.391.0
remote-ssh|ms-|0.115.1
remote-ssh-edit|ms-|0.87.0
remote-wsl|ms-|0.88.5
vscode-remote-extensionpack|ms-|0.26.0
remote-explorer|ms-|0.4.3
remote-server|ms-|1.5.2
nvda-indent-nav-accessibility|Ton|1.0.0
astro-vscode|ast|2.15.4
vscode-eslint|dba|3.0.10
dendron|den|0.124.0
dendron-paste-image|den|1.1.1
prettier-vscode|esb|11.0.0
debugpy|ms-|2024.12.0
python|ms-|2024.20.0
vscode-pylance|ms-|2024.11.3
rust-analyzer|rus|0.4.2193

</details>

**Comment by Neurrone** [comment]
Cc @isidorn

**Comment by Neurrone** [comment]
Latest version is bad, 1.96.0.20241122 [user installer](https://vscode.download.prss.microsoft.com/dbazure/download/insider/90868576241dd25c6c5da64adadc0a09de91a9fe/VSCodeUserSetup-x64-1.96.0-insider.exe)

```
Version: 1.96.0-insider (user setup)
Commit: 90868576241dd25c6c5da64adadc0a09de91a9fe
Date: 2024-11-22T09:56:24.579Z
Electron: 32.2.5
ElectronBuildId: 10579404
Chromium: 128.0.6613.186
Node.js: 20.18.0
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.22631
```

Last good version that doesn't have this bug is the one before that, 1.96.0.20241121 [user installer](https://vscode.download.prss.microsoft.com/dbazure/download/insider/69acde7458f428f0e6869de8915c9dd995cdda1a/VSCodeUserSetup-x64-1.96.0-insider.exe)

```
Version: 1.96.0-insider (user setup)
Commit: 69acde7458f428f0e6869de8915c9dd995cdda1a
Date: 2024-11-21T05:04:38.064Z
Electron: 32.2.5
ElectronBuildId: 10579404
Chromium: 128.0.6613.186
Node.js: 20.18.0
V8: 12.8.374.38-electron.0
OS: Windows_NT x64 10.0.22631
```

**Comment by isidorn** [comment]
Sounds like a regression in insiders. Fyi @meganrogge

**Comment by meganrogge** [comment]
@aiday-mar bisected this to b69589ce385eff52cdfedd7ab89ec8b1fb98ad16

![Image](https://github.com/user-attachments/assets/f54a4fa2-4d37-4629-8fb2-e079d363eca7)

**Comment by aiday-mar** [comment]
Hi thanks for filing this issue. This is a bug with the EditContext enablement. To temporarily fix this issue, you may turn the following setting off: `editor.experimentalEditContextEnabled`. In the meantime, I will look for a fix.

**Comment by Neurrone** [comment]
Given the severity of this issue, it should be turned off by default.

In the same version this issue was introduced, another much more critical regression #234466 was also introduced. Given the setting is called edit context, I wonder if it is related, since these issues occured at the same time.

**Comment by aiday-mar** [comment]
Hi thank you for posting this. I will have a look. I may have to turn off the edit context until the problems are resolved, I will keep you updated.

**Comment by aiday-mar** [comment]
Hi thank you for filing this issue, this seems to be related to the combination of the edit context and the screen reader mode being enabled. I will turn off the edit context by default on Insiders, investigate all the issues and turn it back on later.

**Comment by aiday-mar** [comment]
The PR that reverts the enablement : https://github.com/microsoft/vscode/pull/234638. I will investigate the issues you and @meganrogge filed

**Comment by Neurrone** [comment]
Thanks!
